src/app/view-item/view-item.component.ts

```diff
@@ -15,11 +15,5 @@ interface TodoItem {
 })
 
 export class ViewItemComponent {
-  items: TodoItem = {
-    list: 'example',
-    id: 1,
-    title: 'Example title',
-    description: 'an example list item for the view model',
-    completed: false,
-  };
+  @Input() items: TodoItem;
```

Comment:
Why `ViewItemComponent` has a field `itemS` (plural). Shouldn't it show a single item?
change name to `item`